README.md

```diff
@@ -1,2 +1,44 @@
 # FindYourPet
-Find Your Pet web application.
+Find Your Pet web application Install.
+here is the steps to configurate the project in your pc
+- first you must to install git in your pc.
+# GIT INSTALL 
+* **WINDOWS** :
+1. you will to go at this direccion [press here](https://git-scm.com/download/win") here you be able to install it in your pc.
+2. you will download it.
+3. you must open the executable and follow the steps of installation, when it has finised, you will search in your pc git bash and you will open it.
+
+* **MAC**
+1. you will visit git page in this direccion [press here](https://git-scm.com/download/mac), later you will download git with the steps that you find here.
+2. you will open the executable and will finish the installation
+
+* **LINUX**
+- if you're in a Debian-based distribution like Ubuntu
+*apt-get install git*
+- If you want to install Git on Linux through a binary installer, in general you can do it through the basic package management tool that your distribution brings. If you are in Fedora, you can use yum:
```

Comment:
People doesn't want! They should.

Correct this line with "You should install git on Linux..."